src/styles/calendar.scss

```diff
@@ -0,0 +1,14 @@
+.calendar {
+  margin: auto;
```

Comment:
.calendar is a block, you should't use margin on a block, these are the rules of BEM, i saw the same thing in your catalog code, it's movie form Bartosz from our classes, and he say how to use mix https://www.youtube.com/watch?v=ovKznu-ZHqA